src/platform/macos/config.rs

```diff
@@ -3,7 +3,8 @@ use clap::{App, ArgMatches};
 use config::Config;
 
 /// Platform-specific configuration
-#[derive(Deserialize, Default, Clone)]
+/// Be careful with this, because specifying an unknown field will not cause an error
```

Comment:
What does this comment mean, exactly?

Reply:
The other structs have `#[serde(deny_unknown_fields)]` on them; but because PlatformSpecificConfig is different between platforms (as the name suggests), it doesn't tell you if your configuration has unknown fields and therefore can be confusing when it doesn't work (misspelled field, etc)

Reply:
ahhhhhhhhhhhhhhhh i see what you mean!